admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl

```diff
@@ -28,17 +28,26 @@
     <i class="material-icons">account_circle</i>
   </div>
   <div class="dropdown-menu dropdown-menu-right">
-    <div class="text-center employee_avatar">
-      <img class="avatar rounded-circle" src="{$employee->getImage()}" />
-      <span>{$employee->firstname} {$employee->lastname}</span>
-    </div>
-    <a class="dropdown-item employee-link profile-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}">
-      <i class="material-icons">settings_applications</i>
-      {l s='Your profile'}
+    <div class="employee-wrapper-avatar">
+      
+      <span class="employee_avatar"><img class="avatar rounded-circle" src="{$employee->getImage()}" /></span>
+      <span class="employee_profile">{l s='Welcome back'} {$employee->firstname}</span>
+      <a class="dropdown-item employee-link profile-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}">
+      <i class="material-icons">settings</i>
+      {l s='Your profile' d='Admin.Navigation.Header'}
     </a>
-    <a class="dropdown-item employee-link" id="header_logout" href="{$logout_link|escape:'html':'UTF-8'}">
-      <i class="material-icons">power_settings_new</i>
-      <span>{l s='Sign out'}</span>
+    </div>
+    
+    <p class="divider"></p>
+    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17' d='Admin.Navigation.Header'}"><i class="material-icons">book</i> {l s='Resources'd='Admin.Navigation.Header'}</a>
```

Comment:
I'm not sure URLs should be translated, I think they should be stored somewhere and injected on the templates instead.

Wdyt @LouiseBonnard ?

Reply:
If we plan to make these links localized later, this seems to be a correct implementation in my opinion. We can keep the English link until we get the other langs.

Reply:
I'm wondering about it as well. @Quetzacoalt91, do you think that it needs to be in Crowdin to be localized properly? Links are pretty long...